packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -170,7 +171,7 @@ function ChartsTooltip<T extends ChartSeriesType>(props: ChartsTooltipProps<T>)
   return (
     <NoSsr>
       {popperOpen && (
-        <PopperComponent {...popperProps}>
+        <PopperComponent {...popperProps} className={classes.root}>
```

Comment:
Before that change, the `PopperComponent` has the styleOverride.root, but the `classes.root` was on the `ChartsTooltipPaper`

With that modification both `styleOverrides.root` and `classes.root` are on the same component